src/api/treeTrackerApi.js

```diff
@@ -68,51 +71,75 @@ export default {
       handleError(error);
     }
   },
-  approveCaptureImage(id, morphology, age, captureApprovalTag, speciesId) {
+  approveCaptureImage(capture, morphology, speciesId, captureApprovalTag) {
     try {
-      const query = `${API_ROOT}/api/${getOrganization()}trees/${id}`;
+      log.debug(
+        'approveCaptureImage',
+        capture,
+        morphology,
+        speciesId,
+        captureApprovalTag
+      );
 
-      log.debug(query);
+      // map legacy data to fields for new microservice
+      const newCapture = {
+        id: capture.uuid,
+        reference_id: capture.id,
+        session_id: capture.session_id, // no legacy equivalent
+        grower_account_id: capture.planterId,
+        planting_organization_id: capture.organization_id,
+        device_configuration_id: capture.device_configuration_id, // no legacy equivalent
+        image_url: capture.imageUrl,
+        lat: capture.lat,
+        lon: capture.lon,
+        gps_accuracy: capture.gps_accuracy,
```

Comment:
I would expect this to come through from LoopBack as `gpsAccuracy`, although it's currently omitted in `CAPTURE_FIELDS`. For simplicity, I'd suggest removing the explicit field definition from the `getCaptureImages()` query and just getting everything.

Reply:
I tried that first, but the treetracker api will reject an object with fields that are "not allowed".  I can comment out those fields but some of them are required, so I decided that there would still be work to do but allowing some undefined values for the moment would let us see how it fits with the field-service-api and exactly what would need to be changed.

Reply:
Sorry, you're right – it's in the legacy `trees` table, but not in the LoopBack API.
Similarly to some of the other fields, we can either update the legacy API to include it (undesirable) or have a migration get the missing fields.

Reply:
@Kpoke and I discussed how to handle some of these discrepancies on Tues and figured that we'd best fetch the field-data raw-captures to Verify so that the data fields align.  It's the direction we're moving anyway and it should let each tool mostly work. 

There will be conflicts with data that crosses the legacy and new apis like GrowerDetail and Capture Detail -- the data won't always be found since we're still in progress merging it to the new microservice.